### PR TITLE
Paint mini helicopters

### PR DESCRIPTION
**Pull request by marijnvdwerf**
image

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
@duncanspumpkin The supports of the sloped sections seem to behaving oddly.
image

**Comment by duncanspumpkin** [comment]
I've come across this before. It will take me a little while to remember what is wrong

**Comment by duncanspumpkin** [comment]
There is a small mistake in `metal_a_supports_paint_setup` I'll make the change right now. It should fix your issue.

File: src/ride/gentle/mini_helicopters.c [hunk 1]
```diff
@@ -13,3 +13,369 @@
...
+ */
+TRACK_PAINT_FUNCTION get_track_paint_function_mini_helicopters(int trackType, int direction)
+{
+	switch (trackType) {
+		case TRACK_ELEM_BEGIN_STATION:
+		case TRACK_ELEM_MIDDLE_STATION:
+		case TRACK_ELEM_END_STATION:
+			return paint_mini_helicopters_track_station;
+
+		case TRACK_ELEM_FLAT:
```
Comment (duncanspumpkin) [inline src/ride/gentle/mini_helicopters.c]: Can you keep this is numerical order. It just gets a little confusing when looking down the list.

**Comment by marijnvdwerf** [comment]
@IntelOrca Rebased

**Pull request merged**